src/main/kotlin/net/lumalyte/lg/infrastructure/placeholders/LumaGuildsExpansion.kt

```
@@ -301,6 +306,62 @@ class LumaGuildsExpansion : PlaceholderExpansion(), KoinComponent {
         }
     }
 
+    private fun handleWeeklyQuestPlaceholder(identifier: String, guildId: UUID?): String {
```

detekt (1.23.8)

[warning] The function handleWeeklyQuestPlaceholder appears to be too complex based on Cyclomatic Complexity (complexity: 34). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)